dev-libs/weston/weston-1.12.0.ebuild

```diff
@@ -0,0 +1,150 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+if [[ ${PV} = 9999* ]]; then
+	EGIT_REPO_URI="git://anongit.freedesktop.org/git/wayland/${PN}"
+	GIT_ECLASS="git-r3"
+	EXPERIMENTAL="true"
+fi
+VIRTUALX_REQUIRED="test"
+RESTRICT="test"
+
+inherit autotools readme.gentoo-r1 toolchain-funcs virtualx $GIT_ECLASS
+
+DESCRIPTION="Wayland reference compositor"
+HOMEPAGE="https://wayland.freedesktop.org/"
+
+if [[ $PV = 9999* ]]; then
+	SRC_URI="${SRC_PATCHES}"
+	KEYWORDS=""
+else
+	SRC_URI="https://wayland.freedesktop.org/releases/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~x86 ~arm-linux"
+fi
+
+LICENSE="MIT CC-BY-SA-3.0"
+SLOT="0"
+
+IUSE_VIDEO_CARDS="video_cards_intel video_cards_v4l"
+IUSE="colord dbus +drm editor examples fbdev +gles2 headless ivi jpeg lcms rdp +resize-optimization +launch screen-sharing static-libs +suid systemd test unwind wayland-compositor webp +X xwayland ${IUSE_VIDEO_CARDS}"
+
+REQUIRED_USE="
+	drm? ( gles2 )
+	screen-sharing? ( rdp )
+	systemd? ( dbus )
+	test? ( X )
+	wayland-compositor? ( gles2 )
+"
+
+RDEPEND="
+	>=dev-libs/libinput-0.8.0
+	>=dev-libs/wayland-1.12.0
+	>=dev-libs/wayland-protocols-1.2
+	lcms? ( media-libs/lcms:2 )
```

Comment:
dependencies should be sorted, with USE-conditional dependencies (also sorted) at the bottom. that would be a nice-to-have ;)